File: JAVA/09_Math2/BOJ_01_1978_소수찾기.java
import java.io.IOException;
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        int[] tmp = new int[n];
        int sum = 0;
        StringTokenizer st = new StringTokenizer(br.readLine(), " ");
        for (int i = 0; i < n; i++) {
            tmp[i] = Integer.parseInt(st.nextToken());
            int cnt = 0;
            for (int j = 1; j <= tmp[i]; j++) {
                if (tmp[i] % j == 0) {
                    cnt++;
                }
            }
            if (cnt == 2) {
                sum++;
            }
        }
        System.out.println(sum);
    }
}

File: C/09_Math2/BOJ_01_1978_소수찾기.c
#include <stdio.h>
main() {
	int n, tmp, sum = 0;
	scanf("%d", &n);
	while (n--) {
		scanf("%d", &tmp);
		int cnt = 0;
		for (int i = 0; i < tmp; i++) {
			if (tmp % (i + 1) == 0) {
				cnt++;
			}
		}
		if (cnt == 2) {
			sum++;
		}
	}
	printf("%d", sum);
}